indico/modules/events/templates/admin/autolinker.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'layout/admin_page.html' %}
+{% from 'events/admin/_event_label_list.html' import render_event_label_list %}
+
+{% block title %}
+    {% trans %}Auto-linked Patterns{% endtrans %}
+{% endblock %}
+
+{% block description %}
+    <div class="ui icon purple message">
+        <i class="lab icon"></i>
+        <div class="content">
+            <div class="header">{% trans %}Experimental feature{% endtrans %}</div>
+            <p>
+                {% trans -%}
+                    Make sure you understand what you're doing or your notes may end up broken!
+                {%- endtrans %}
+            </p>
+        </div>
+    </div>
+    <p>
+        {% trans -%}
+            Here you can define text patterns which are automatically linked e.g. to ticketing or code
+            repository systems, from within your notes.
+        {%- endtrans %}
+    </p>
+    <div class="ui icon message">
+        <div class="content">
+            <div class="header">{% trans %}Example{% endtrans %}</div>
+            <code>
+                gh:([\d\w\-_\.]+)/([\d\w\-_\.]+):(\d+) → https://github.com/{1}/{2}/issues/{3}
```

Comment:
What about using `\b` around it? Almost all regexes here should probably be using word boundaries anyway.
I'd also use `#` instead of `:` here since that more closely resembles GitHub's issue linking notation

Reply:
![image](https://github.com/indico/indico/assets/179599/e92ada29-eda1-4e06-96fd-037ebed55e28)

Might even be worth showing a warning if the regex does not have boundaries.

Reply:
Or always add `\b...\b` around the regex? Any cases where we would not want this?

https://regex101.com/r/AGF5Vk/1

Reply:
Counter-example: `google:(?:(?:"([^"]+)")|([^\s]+))`. I know it's not super-common for those patterns to allow spaces, but maybe we should leave the choice to the regex implementor?

Reply:
hm, maybe at least mention that usually boundaries should usually be added and include it in the example?